tools/downloader/topologies.py

```diff
@@ -0,0 +1,99 @@
+import os
+import argparse
+
+from pathlib import Path
+
+import common
+import downloader
+import converter
+
+class Topology:     
+    '''
+    {}
+    Model task type: {}
+
+    License: {}
+    '''
+
+    def __init__(self, model_name, *, precision=None, download_dir=None):
+        '''
+        Downloads target model. If the precision doesn't set, loads all the available models.
+        If there is already downloaded model - retrieves the model from cache.
+        Creates an object which stores information about the downloaded model.
+
+        Parameters
+        ----------
+            model_name
+                Target model name.
+            precision
+                Target model precision. By default is None.
+            download_dir
+                Model download directory.
+        '''
+        if not download_dir: 
+            download_dir = Path(Path('.').resolve() / 'models')
+        cache_dir = Path(Path(download_dir) / '.cache')
+
+        parser = argparse.ArgumentParser()
+        args = argparse.Namespace(all=False, list=None, name=model_name, print_all=False)   
+        topology = common.load_models_from_args(parser, args)[0]
+
+        model_dir = download_dir / topology.subdirectory
+
+        flags = ['--name=' + model_name,
+                 '--output_dir=' + str(download_dir).replace('\\', '\\\\'), 
+                 '--cache_dir=' + str(cache_dir).replace('\\', '\\\\')]
```

Comment:
What's the purpose of all these replacements?

Reply:
It goes from the closed PR. I just remember that there were some issues with Windows path so `\` should be replaced to `\\`. But we cannot write just `.replace('\', '\\')`

Reply:
I remember it from the previous PR, but I didn't get a satisfactory explanation then either. I'd like to know something more specific than "some issues".

Reply:
We needn't this replace for Windows, because user can't write like:
```python
ssd = omz.Model('mobilenet-ssd', precisions='FP16', download_dir='C:\User\model')
```
it causes
```
SyntaxError: (unicode error) 'unicodeescape' codec can't decode bytes in position 2-3: truncated \UXXXXXXXX escape
```

We should do:
```python
ssd = omz.Model('mobilenet-ssd', precisions='FP16', download_dir='C:\\User\\model')
```
or
```python
ssd = omz.Model('mobilenet-ssd', precisions='FP16', download_dir='C:/User/model')
```